config/initializers/statsd.rb

```diff
@@ -64,7 +64,7 @@
 
 # init Vet360
 Vet360::Exceptions::Parser.instance.known_keys.each do |key|
-  StatsD.increment("#{Vet360::Service::STATSD_KEY_PREFIX}.exceptions.#{key}", 0)
+  StatsD.increment(Vet360::Service::STATSD_KEY_PREFIX, 0, tags: ["exceptions:#{key}"])
```

Comment:
I think we should rename the metric to `#{Vet360::Service::STATSD_KEY_PREFIX}.exceptions` and change the tag name to `exception`. That would make it (hopefully) more obvious to the next person coming along what the metric is and how to break it down.